Product Harmonization

adriens opened the pull request:
👉  See https://github.com/endoflife-date/endoflife.date/issues/2016 for more details

adriens committed:
Product harmonization for Red Hat  Ansible

adriens committed:
product harmoization Microsoft

adriens committed:
Product harmonization Hashicorp

adriens committed:
Product harmonization Microsoft

adriens committed:
product harmonization Microsoft

adriens committed:
product harmonization Hashicorp

adriens committed:
Product harmonizaation Apple

adriens committed:
Product harmonization

adriens committed:
Typo Apple product harminzation

adriens committed:
Product harmonzation Oracle MySQL

adriens committed:
Product harmonization Microsoft

adriens committed:
Product harmonization VMware RabbitMQ

adriens committed:
product harmonization Hashicorp Terraform

adriens committed:
Product harmonization Microsoft  visual studio

adriens committed:
Product harmonization Microsoft  Windows

adriens committed:
Product harmonization Microsoft Embedded

adriens committed:
Product harmonization

usta committed:
Update azure-devops.md

captn3m0 commented on products/ansible.md:
```diff
@@ -1,6 +1,6 @@
 ---
 permalink: /ansible
-title: Ansible
+title: Red Hat Ansible
```
I think this has the same problem as opensearch. RH Ansible is [something else](https://www.redhat.com/en/technologies/management/ansible), which we don't track.

marcwrobel replied:
Agree with @captn3m0, but not for the same reasons. The "something else" is "Red Hat Ansible Automation Platform", not just "Red Hat Ansible".

Having said that Red Hat never refers to Ansible using "Red Hat Ansible", it is always just "Ansible" (see https://www.ansible.com/integrations/infrastructure/red-hat for example).

captn3m0 commented on products/mysql.md:
```diff
@@ -1,5 +1,5 @@
 ---
-title: MySQL
+title: Oracle MySQL
```
Have never heard of this phrasing. -1

usta replied:
i know it i am not the only one :D 
https://github.com/endoflife-date/endoflife.date/pull/2036#issuecomment-1350603622

adriens replied:
:trollface:

captn3m0 commented on products/rabbitmq.md:
```diff
@@ -1,5 +1,5 @@
 ---
-title: RabbitMQ
+title: VMware RabbitMQ
```
-1

> VMware offers a [range of commercial offerings for RabbitMQ](https://tanzu.vmware.com/rabbitmq). This includes a distribution called [VMware RabbitMQ](https://network.pivotal.io/products/p-rabbitmq-for-kubernetes/info) that deploys on Kubernetes or your container Runtime, as well as a version that deploys in [VMware Tanzu Application Service](https://tanzu.vmware.com/tanzu).

VMWare RabbitMQ is something else.

usta replied:
i know it i am not the only one :D 
https://github.com/endoflife-date/endoflife.date/pull/2036#issuecomment-1350603622

usta replied:
actually it is same with commercial license [ just like oracle java ]   but still my 2 cents is for not having a company name infront of this product

captn3m0 reviewed (changes requested):
Suggest undoing some confusing changes. Simple ones (Microsoft, Hashicorp) should be good.

Let's look at tags instead for such usecases.

adriens reviewed (commented):
Sure :+1:

adriens reviewed (commented):
Sure :+1:

adriens committed:
Ansible vs. Red Hat Ansible

cf https://github.com/endoflife-date/endoflife.date/pull/2036#pullrequestreview-1216962745

adriens committed:
Rabbit MQ vs. VMWare RabbitMQ

adriens committed:
Azure DevOps vs MS Azure DevOps

captn3m0 committed:
[mysql] Stick to MySQL

captn3m0 reviewed (approved):
LGTM. Squash Merging.
